src/plugins/exception-helpers.ts

```diff
@@ -11,32 +15,32 @@ export default function exceptionHelpersPlugin (app) {
   app.addHelper('websocketException', websocketException);
 }
 
-function exception (ctx, error) {
+function exception (ctx: MojoContext, error: Error): boolean {
   if (ctx.isWebSocket) return ctx.websocketException(error);
   return ctx.httpException(error);
 }
 
-async function htmlException (ctx, error) {
-  ctx.stash.exception = error;
+async function htmlException (ctx: MojoContext, error: Error): Promise<boolean> {
+  (ctx.stash as MojoStash).exception = error;
 
-  const mode = ctx.app.mode;
-  if (await ctx.render({view: `exception.${mode}`, maybe: true, status: 500}) === true) return;
-  if (await ctx.render({view: 'exception', maybe: true, status: 500}) === true) return;
+  const mode: string = ctx.app.mode;
```

Comment:
That seems odd, `app.mode` is already typed. Does it detect `ctx.app` correctly?